test/stdlib/SwiftObjectNSObject.swift

```diff
@@ -16,7 +16,7 @@
 // RUN: %target-build-swift %s -g -I %S/Inputs/SwiftObjectNSObject/ -Xlinker %t/SwiftObjectNSObject.o -o %t/SwiftObjectNSObject
 // RUN: %target-codesign %t/SwiftObjectNSObject
 // RUN: %target-run %t/SwiftObjectNSObject 2> %t/log.txt
-// RUN: cat %t/log.txt 1>&2
```

Comment:
@tbkka Do you remember what the purpose is of this `cat`?

Reply:
> That looks like something done in frustration because test results weren't appearing on the console.

Once we enable the internal shell that output should hopefully be a lot clearer :)